fix(parser): internal compiler error for errors after the end of file (#2485)
tact-lang/tact: dev-docs/CHANGELOG.md

- Optimized `Context().sender` to use `sender()` function for better gas efficiency: PR [#2427](https://github.com/tact-lang/tact/pull/2427)
- [fix] Ternary operator with struct and null: PR [#2432](https://github.com/tact-lang/tact/pull/2432)

> - [fix] Fix internal compiler error for an incomplete return statement: PR [#2485](https://github.com/tact-lang/tact/pull/2485)

Reviewer: ```suggestion
- [fix] The parser does not throw an internal compiler error if there is the closing brace for the function body is missing: PR [#2485](https://github.com/tact-lang/tact/pull/2485)
```

> - [fix] The parser does not throw an internal compiler error if error is reported after the end of file: PR [#2485](https://github.com/tact-lang/tact/pull/2485)

Reviewer: ```suggestion
- [fix] The parser does not throw an internal compiler error if the error is reported after the end of the file: PR [#2485](https://github.com/tact-lang/tact/pull/2485)
```

> - [fix] The parser does not throw an internal compiler error if there is the closing brace for the function body is missing: PR [#2485](https://github.com/tact-lang/tact/pull/2485)

Reviewer 2: ```suggestion
- [fix] The parser does not throw an internal compiler error if error is reported after the end of file: PR [#2485](https://github.com/tact-lang/tact/pull/2485)
```
